package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "leaflet-tilelayer-geojson",
+  "version": "1.0.4",
+  "description": "Leaflet plugin to render GeoJSON data over tile layer",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/glenrobertson/leaflet-tilelayer-geojson.git"
+  },
+  "keywords": [
+    "Leaflet",
+    "leaflet.js",
+    "tile",
+    "tilelayer",
+    "geojson"
+  ],
+  "author": "glenrobertson",
+  "license": "ISC",
```

Comment:
What's the detail about BSD?
[license](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json)
Thanks.